src/common/utils/gpo.h

```diff
@@ -59,6 +59,7 @@ namespace powertoys_gpo {
     const std::wstring POLICY_CONFIGURE_ENABLED_ENVIRONMENT_VARIABLES = L"ConfigureEnabledUtilityEnvironmentVariables";
     const std::wstring POLICY_CONFIGURE_ENABLED_QOI_PREVIEW = L"ConfigureEnabledUtilityFileExplorerQOIPreview";
     const std::wstring POLICY_CONFIGURE_ENABLED_QOI_THUMBNAILS = L"ConfigureEnabledUtilityFileExplorerQOIThumbnails";
+    const std::wstring POLICY_CONFIGURE_ENABLED_NEW = L"ConfigureEnabledUtilityNewPlus";
```

Comment:
The same as on the policy name. This is confusing. Please rename to `NewPlus`.

Reply:
Thank you! I've finally gotten around to renaming to NewPlus throughout. Thanks again.